feat(locale): add country code for en_CA & fr_CA

alixlahuec opened the pull request:
### Description

Configure `country_code` for the `en_CA` and `fr_CA` locales.

### Checklist

✅  I've ran `pnpm run preflight`
✅  I've read https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md

alixlahuec committed:
update en_CA, fr_CA locales

xDivisionByZerox commented:
Thank you for your contribution @alixlahuec 🙏❤️. 
I see that this was your first contribution to our project.
Would you mind answering some questions regarding your contribution? This way we can understand the potential struggles or confusions new contributors might have when getting into the project. If you don't want to answer, please ignore this comment:

1. What did motivate you to do your contribution?
1. Did you read our documentation regarding contributing before starting to work on your contribution?
1. How clear was the process of working on your contribution (before the PR) for you? Was there ever a time while working in the project where you didn't know what to do? If so how did you solve your problem?
1. How clear was the process of the code review (PR) for you? Was there ever a time you felt unsure about the current situation?
1. Anything else you'd like to share with the Faker team?

alixlahuec commented:
@xDivisionByZerox  Sure, I'd be happy to!

> 1. What did motivate you to do your contribution?

My team uses Faker to easily generate test data, we're Canada-based and recently came across some use cases where the locale we used with Faker mattered (we'd been using just `en`). We recently shipped some work related to addresses and postal codes, and realized our test data was failing validation because `en` uses US postal codes format by default & we just had never really realized this. I switched us to use `en_CA` which just makes so much more sense for us, and I noticed it was really easy to make the locale data more complete there so I did a quick PR!

> 2. Did you read our documentation regarding contributing before starting to work on your contribution?

Yes ✅ 

> 3. How clear was the process of working on your contribution (before the PR) for you? Was there ever a time while working in the project where you didn't know what to do? If so how did you solve your problem?

It was pretty clear overall, I think there's a few points where I wasn't sure and checked previous contributions / ran commands to be confident I'd done what was needed:
- I wasn't sure if I needed to write tests.
- I wasn't sure if I needed to do some kind of validation. I did end up doing it by opening up the Faker docs for the PR, and using the console to create a Faker instance with the locales I was updating, then generated the data kind I updated (country code) to make sure it was giving the expected result.
- I wasn't sure which locale updates require running the generate command or not, so in doubt I ran it & I got no diffs - I looked at previous PRs that were similar to mine and the diffs were similar, so it looked correct to me

> 4. How clear was the process of the code review (PR) for you? Was there ever a time you felt unsure about the current situation?

Super clear (super fast!) 💯  I wasn't too sure how the Faker team organizes releases (what goes into what) but I wasn't worried about it!

> 5. Anything else you'd like to share with the Faker team?

Thanks for your work! We love using Faker, and I'll be happy to contribute back to the project when I can

xDivisionByZerox commented:
Thank you for your feedback.  It's awesome to see that you were using the documentation preview to validate your changes - very creative.

>  I wasn't too sure how the Faker team organizes releases [...]

I guess that you might be interested when you will actually be able to use your changes. The faker team tries to do monthly minor releases at the start of a month. So you should able to make your of your changes in Faker v9.8.0 in about 3 weeks. 🙂
